Sources/Rendering/Core/Texture/index.js

```diff
@@ -100,20 +100,155 @@ function vtkTexture(publicAPI, model) {
     const dimensionality = (width > 1) + (height > 1) + (depth > 1);
     return dimensionality;
   };
+
+  publicAPI.getImageData = () => {
```

Comment:
Probably need to call this getJSImageData as people who use VTK will assume it is returning vtkImageData

Reply:
JsImageData is already a property in texture, is there something else you'd prefer it to be called?

Reply:
getInputAsJSImageData

Reply:
though it doesn't seem to handle the vtkImageData case, is that intended?

Reply:
I implemented it mostly so I could easily get raw image data from canvas and html images while testing mipmap generation.  It's not being used right now but I thought it would be a nice convenience feature for the two types of images I intended it for since they're pretty annoying to get data from